### PR TITLE
symlink runme.sh to be across all targets

### PR DESCRIPTION
**Pull request by shellymiron**
This PR will refactor the way a target runs, by linking the original `runme.sh` to each target folder that has the prefix `vmware_ops_` instead of recreating each time the `runme.sh` when writing the integration tests.

note: I did not remove the `runme.sh` from targets that were not modified to be suitable to run on the vcenter setup,
once those tests will be enhanced, we will delete this script from them.

### CODE REVIEW

**Review by anna-savina — APPROVED** [review]

File: prepare_symlinks.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+- name: Create symlink to the runme.sh script in specific target directories
```
Comment (machacekondra) [inline prepare_symlinks.yml:1]: Please add this script to something like "tools" directory. If I understand it correclty it's for manual execution only, right?

**Review by elsapassaro — APPROVED** [review]

File: prepare_symlinks.yml (hunk 1 above)
Comment (shellymiron) [reply prepare_symlinks.yml:1]: I'm using this script for CI, if you will look into the makefile you will see I'm adding it there:
```
.PHONY: prepare_symlinks
prepare_symlinks:
	ansible-playbook prepare_symlinks.yml
```
this approach actually make the manual testing to require creating this `runme.sh` manually for the target

Comment (machacekondra) [reply prepare_symlinks.yml:1]: Makes complete sense. I would just move the playbook outside of root directory.

Comment (shellymiron) [reply prepare_symlinks.yml:1]: sure, on it

Comment (shellymiron) [reply prepare_symlinks.yml]: done

**Comment by shellymiron** [comment]
@machacekondra good to be merged? :D

**Review by machacekondra — APPROVED** [review]

**Pull request merged**